reference/price_lists.v3.yml

```diff
@@ -2032,13 +2184,6 @@ paths:
                   date_modified: '2022-09-18T13:18:15Z'
                   currency: eur
                   product_id: 185
-                meta:
```

Comment:
Was this removed because we're looking at the singleton endpoint here?

Reply:
@megdesko that was a mistake. I added meta back in. 
When I tested this endpoint, the response looks as follows:

{
    "data": {
        "price_list_id": 8,
        "variant_id": 205,
        "price": 3.99,
        "sale_price": 0.1,
        "retail_price": 0.1,
        "map_price": 0.1,
        "calculated_price": 0.1,
        "date_created": "2025-03-17T15:47:48Z",
        "date_modified": "2025-03-17T15:52:25Z",
        "currency": "usd",
        "product_id": 141
    },
    "meta": {}
}